src/analysis/souffle/operations.dl

```diff
@@ -48,11 +48,32 @@ operationToPartialOperandLists([owner, op, result, operandList, attrs], operandL
 operationToPartialOperandLists(operation, tail) :-
   operationToPartialOperandLists(operation, [_, tail]).
 
-// Map full operations to operands.
-.decl operationToOperands(operation:Operation, operand: AccessPath)
+// `listLength` is the length of the `operandList`.
+.decl operandListLength(operandList: OperandList, listLength: number)
```

Comment:
Should I just drop #548 and you can fix the names in this PR? Or, should I try to merge it in?

Reply:
Maybe that would be best. If we both submitted, there would be some significant conflicts that would just duplicate the work anyway.

Reply:
Sounds good. I will close the other PR.